smart-contract/lottery/src/lottery.rs

```diff
@@ -382,177 +284,187 @@ impl Lottery {
         u64::from_be_bytes(hashed_data[..8].try_into().unwrap()) % 100
```

Comment:
Please add this link to todo https://docs.rs/casper-contract/latest/casper_contract/contract_api/runtime/fn.random_bytes.html